proto/tendermint/farming/v1beta1/farming.proto

```diff
@@ -180,3 +180,13 @@ message OutstandingRewards {
   repeated cosmos.base.v1beta1.DecCoin rewards = 1
       [(gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.DecCoins", (gogoproto.nullable) = false];
 }
+
+// AddressType enumerates the available types of a address.
+enum AddressType {
```

Comment:
How about defining address type as Go `type`, since other proto messages don't use this proto `enum`.

For example:
```go
type AddressType int

const (
        AddressType32Bytes AddressType = iota + 1
        AddressType20Bytes
)
```

Reply:
`AddressType` is used as query options on [Budget](https://github.com/tendermint/budget/pull/90) module, and although it can be imported and used in type from Budget, they were deliberately declared redundant in the same way for module independence and consistency.

Reply:
It has been decided to remain the same for consistency.